### ISSUE TITLE
Dataset page navigation discrepancies (viewing a dataset from a Group)

### ISSUE THREAD

**Issue by sheilamv** [bcgov/ckan-ui#230]
As a consumer of data (Public or Government), I would expect that the navigation from a Group list would take me back to the list of datasets within the selected group.

The steps to reproduce can be found [here](https://bcgov.sharepoint.com/:x:/r/teams/00108/Shared%20Documents/PROD_BCDC/UIUX%20User%20Acceptance%20Testing/Templates/EDC_UAT_NewUI_SMV.xlsx?d=wa02da30961264f17b72e240c98a35074&csf=1&web=1&e=dYac8c), Public (No Login) tab for test case 10. 

**Highlights**:

- From this page (https://toyger.data.gov.bc.ca/group/bc-government-api-registry), select the first dataset in the list (https://toyger.data.gov.bc.ca/dataset/bc-gov-news-api-service).  One would think that a 'back' navigation would take the user to the list of dataset in the group, not the list of ALL datasets (https://toyger.data.gov.bc.ca/datasets) - **FAIL**

- _Share_ link should be renamed  to _Share link_ or _Copy link_

- Clicking on a Contact's Organization should be presented as a popup window.  Taking the user to the organization page makes it hard for the user to come back to the dataset they are viewing.

- Displayed label should not be displayed - the checkbox is to determine if the contact is displayed or not to a user not logged in.  **Checked** = Contact record displayed to unlogged in user **Unchecked** = Contact record NOT displayed to unlogged in user
(Used this dataset:  data.gov.bc.ca/dataset/bc-wildfire-prescribed-fire-locations)

- **Suggestion**:  if there is no supporting information, the system should display text to inform the user that there is no data - "No data provided"

- **Suggestion**:  Under More Information, there is no need to display the URL label.  It is redundant, especially when the data is explicit.  The data provided will always be urls. 

_Contact redesign consideration_:

- **Design recommendation**:  Move the contacts over to the right hand side either above or below the Resources.  It is not necessary to give this much real estate to Contacts.  It can even be up at the top to the right of Groups link and be a popup window like :earn more about this organization.

- **Suggestion**:  link text = Contacts for this dataset

- **Suggestion**:  rather than list each person individually with their role, group users by Roles to reduce redundancy and display as First Name Last Name | email address | Organization

**Comment by BrandonSharratt** [comment]
_From this page (https://toyger.data.gov.bc.ca/group/bc-government-api-registry), select the first dataset in the list (https://toyger.data.gov.bc.ca/dataset/bc-gov-news-api-service). One would think that a 'back' navigation would take the user to the list of dataset in the group, not the list of ALL datasets (https://toyger.data.gov.bc.ca/datasets)._
I have made the UI remember the last list page you were on and a small amount of nav history, so it will now take you back to where you would expect

_Share link should be renamed to Share link or Copy link_
This change was made by the UI designers so won't fix

_Displayed label should not be displayed - the checkbox is to determine if the contact is displayed or not to a user not logged in. Checked = Contact record displayed to unlogged in user Unchecked = Contact record NOT displayed to unlogged in user_
The functionality was already fixed in a different ticket, it is now hidden on the view page.

Most of your suggestions are not technically feasible. Because contacts are the same as any other field in the schema. However, I will assign this ticket to @yuisotozaki so that he sees them.
